front/app/modules/commercial/insights/admin/components/ShowHiddenNodes/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { colors } from 'utils/styleUtils';
+
+// styles
+import styled from 'styled-components';
+
+// components
+import { Box, Icon } from '@citizenlab/cl2-component-library';
+import { TooltipContentList } from 'modules/commercial/insights/admin/components/StyledTextComponents';
+import Tippy from '@tippyjs/react';
+
+// intl
+
+import { FormattedMessage } from 'react-intl';
+import messages from '../../containers/Insights/messages';
+
+const TooltipIcon = styled(Icon)`
```

Comment:
No need for a styled component - these can be passed directly as props to `Icon`

Reply:
how would that be? there is no style prop in the Icon component and i can only find examples of what i did though the codebase.

Reply:
You can pass `width` `height` `fill` and `margin` directly as props to `Icon` :)

Reply:
oh right the `SvgProps`, missed it! you can go back to sleep now, sorry 😅